### PR TITLE
WD: Audinux project

### PR DESCRIPTION
**Pull request by rosewms**
Atlantic 2024 Writing Day project, Audinux.

Steps before publishing:

- [x] Staff review of project to confirm Writing Day suitability
- [x] WD project lead/organizer review to ensure project is described properly
- [ ] Final staff review before publishing 

----
📚 Documentation preview 📚: https://writethedocs-www--2200.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/atlantic/2024/writing-day.rst [hunk 1]
```diff
@@ -127,3 +127,43 @@ Project list
...
+
+
+
+
+
+
+
+
+
+
```
Comment (plaindocs) [inline docs/conf/atlantic/2024/writing-day.rst]:
```suggestion
```

File: docs/conf/atlantic/2024/writing-day.rst [hunk 2]
```diff
@@ -127,3 +127,43 @@ Project list
...
+Audinix - Stash and Publish a User Guide (Session 1)
```
Comment (plaindocs) [inline docs/conf/atlantic/2024/writing-day.rst]: Stash???

File: docs/conf/atlantic/2024/writing-day.rst [hunk 3]
```diff
@@ -127,3 +127,43 @@ Project list
...
+
+- Introduction to Audinux
+- How to find right Audio/Music packages for your creative work
+- Plugins vs hardware outboard gear
+- Initial configuration
+- Plugin host (carla) and format in Linux
+- Use cases: Home recording
+- How to find subpackages that complement plugins and install subpackages
+- How to load plugins to DAW (Digital Audio Workstation) and OBS Studio
+- Rack and patchbay
```
Comment (plaindocs) [inline docs/conf/atlantic/2024/writing-day.rst]: This  is unrealistic, but sure :-D

**Review by plaindocs — APPROVED** [review]

**Review by mxsasha — APPROVED** [review]
Agreed with Sam, seems like a lot for one session

File: docs/conf/atlantic/2024/writing-day.rst (hunk 2 above)
Comment (hankuoffroad) [reply docs/conf/atlantic/2024/writing-day.rst:146]: Sorry, I've sprinkled a Git terminology 'Git stash', meaning temporarily saving the content in staging area. I'll be explaining that during introduction if that's ok. In particular, a new article requires step by step approaches of building paragraphs and setting them aside, picking some for final publication.

File: docs/conf/atlantic/2024/writing-day.rst [hunk 4]
```diff
@@ -127,3 +127,43 @@ Project list
...
+
+`Audinux <https://audinux.github.io/>`__ is a repository for FOSS music applications and plugins. We want to empower creative people and help them use Audinux to it's full potential! To do that, we need documentarians to help us create a user guide.
+
+Our Writing Day goal is to work together and create a user guide for Audinux. Creating a user guide will positively impact our users and encourage new users.
+
+Our plan is to create new documentation pages within the 'User Guide' templates. This means you can contribute using the GitHub web editor or using whatever Git-workflow from your favorite local writing setup and text editor. If you are unfamiliar with GitHub, we recommend using the `hackMD <https://hackmd.io/>`__ integration with GitHub.
+
+Tentative outline and table of contents:
+
+- Introduction to Audinux
```
Comment (hankuoffroad) [inline docs/conf/atlantic/2024/writing-day.rst]: Session 1
Introduction to Audinux
Intended audience
Content type
Tools for publication
End to end processes
Template: Initial configuration for music creation (Instrument: Electric guitar)
Audio tools

Session 2
How to find right Audio/Music packages for your creative work
Audio Plug-ins

Topics to choose from;
How to find subpackages that complement plugins and install subpackages
How to load plugins to DAW (Digital Audio Workstation) and OBS Studio
Rack and patchbay
Use cases: Home recording

File: docs/conf/atlantic/2024/writing-day.rst
Comment (hankuoffroad) [inline docs/conf/atlantic/2024/writing-day.rst:1]: I've updated it on the Google Forms as well: https://docs.google.com/forms/d/e/1FAIpQLSe4uacCEfqIGMcjnO6UDA6ss0vUAKsSBsXg3-WhdjI0tHBqmA/viewform?edit2=2_ABaOnudDYJ7h1YGMNvs7ED31_W0irQWwptOL3Ct_Idwsma3RByoDrbALuZlvg5tVT4tFxlE

I've invited the maintainer of (Audinux GitHub ID: ycollet) for a joint review with me.

Comment (hankuoffroad) [inline docs/conf/atlantic/2024/writing-day.rst:1]: Here is the project repository for participants to make PR: https://github.com/audinux/audinux.github.io/tree/main/pages/user_guide

**Review by ycollet — APPROVED** [review]
Looks Good To Me !

**Comment by hankuoffroad** [comment]
> Looks Good To Me !

I have created issue tickets in Audinux repo, so attendees can self-assign it and work whenever they're available during the day asynchronously. Due to time difference between US East and Europe, I would monitor issue tickets at Audinux repository.

Ticket:
https://github.com/audinux/audinux.github.io/issues/created_by/hankuoffroad

**Review by hankuoffroad — CHANGES_REQUESTED** [review]
Submitting the changes for review. Thanks for your help.

File: docs/conf/atlantic/2024/writing-day.rst (hunk 2 above)
Comment (hankuoffroad) [inline docs/conf/atlantic/2024/writing-day.rst:146]: Audinux - Plan and Publish a User Guide

File: docs/conf/atlantic/2024/writing-day.rst [hunk 5]
```diff
@@ -127,3 +127,43 @@ Project list
...
+
+Join project organizer, Hank Lee, during the first Writing Day session.
+
+`Audinux <https://audinux.github.io/>`__ is a repository for FOSS music applications and plugins. We want to empower creative people and help them use Audinux to it's full potential! To do that, we need documentarians to help us create a user guide.
+
+Our Writing Day goal is to work together and create a user guide for Audinux. Creating a user guide will positively impact our users and encourage new users.
+
+Our plan is to create new documentation pages within the 'User Guide' templates. This means you can contribute using the GitHub web editor or using whatever Git-workflow from your favorite local writing setup and text editor. If you are unfamiliar with GitHub, we recommend using the `hackMD <https://hackmd.io/>`__ integration with GitHub.
+
+Tentative outline and table of contents:
```
Comment (hankuoffroad) [inline docs/conf/atlantic/2024/writing-day.rst]: Session topics

File: docs/conf/atlantic/2024/writing-day.rst (hunk 2 above)
Comment (hankuoffroad) [inline docs/conf/atlantic/2024/writing-day.rst]: Audinux - Plan and Publish a User Guide

File: docs/conf/atlantic/2024/writing-day.rst [hunk 6]
```diff
@@ -127,3 +127,43 @@ Project list
...
+Audinix - Stash and Publish a User Guide (Session 1)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Join project organizer, Hank Lee, during the first Writing Day session.
+
+`Audinux <https://audinux.github.io/>`__ is a repository for FOSS music applications and plugins. We want to empower creative people and help them use Audinux to it's full potential! To do that, we need documentarians to help us create a user guide.
+
+Our Writing Day goal is to work together and create a user guide for Audinux. Creating a user guide will positively impact our users and encourage new users.
+
+Our plan is to create new documentation pages within the 'User Guide' templates. This means you can contribute using the GitHub web editor or using whatever Git-workflow from your favorite local writing setup and text editor. If you are unfamiliar with GitHub, we recommend using the `hackMD <https://hackmd.io/>`__ integration with GitHub.
```
Comment (hankuoffroad) [inline docs/conf/atlantic/2024/writing-day.rst]: Delete: If you are unfamiliar with GitHub, we recommend using the `hackMD <https://hackmd.io/>`__ integration with GitHub.

File: docs/conf/atlantic/2024/writing-day.rst [hunk 7]
```diff
@@ -143,27 +143,41 @@ Submit your project using our `Writing Day project form <https://forms.gle/uTkWH
 
 
 
-Audinix - Stash and Publish a User Guide (Session 1)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Audinix - Plan and Publish a User Guide (All day)
```
Comment (hankuoffroad) [inline docs/conf/atlantic/2024/writing-day.rst]: Audinux

**Review by hankuoffroad — CHANGES_REQUESTED** [review]

File: docs/conf/atlantic/2024/writing-day.rst [hunk 8]
```diff
@@ -152,3 +152,42 @@ Agenda:
...
+
+Join project organizer, Hank Lee, during Writing Day.
+
+Plan and Publish User Guide for Audinux, a repository for FOSS music applications and plugins
+
+`Audinux <https://audinux.github.io/>`__  is a repository for FOSS music applications and plugins. We want to empower creative people and help them use Audinux to it's full potential! To do that, we need documentarians to help us create a user guide.
+
+Our Writing Day goal is to work together and create a user guide for Audinux. Creating a user guide will positively impact our users and encourage new users.
+
+Our plan is to create new documentation pages within the 'User Guide' templates, check out our `project repository <https://github.com/audinux/audinux.github.io/tree/main/pages/user_guide)>`__. This means you can contribute using the GitHub web editor or using whatever Git-workflow from your favorite local writing setup and text editor.
```
Comment (hankuoffroad) [inline docs/conf/atlantic/2024/writing-day.rst]: check out our `project repository <https://github.com/audinux/audinux.github.io/tree/main/pages/user_guide

// could you remove ) at the end of sentence? Thanks.

**Review by hankuoffroad — CHANGES_REQUESTED** [review]

**Comment by rosewms** [comment]
@hankuoffroad I am going to get this live. If any additional edits are needed, please submit a PR. Thanks!

**Pull request merged**